### ISSUE TITLE
How to store information from PeriodicDataFrame

### ISSUE THREAD

**Issue by rebeckaflynn1** [python-streamz/streamz#451]
i am new to coding, so apologies for asking this. I am looking to store the data that comes in through PeriodicDataFrame. My project is to intake Barometric Pressure every 30 seconds, and then later tell me if there is a change in pressure by a certain amount. In order to do the second part of this, I would need to store the data frame instead of keeping just the latest. How can this be done with streamz, or is it not possible?

**Comment by martindurant** [comment]
Yes, this is a design goal of streamz: nodes can have state, and consider new data in the context of that state. There are several example classes of this (see the code), or maybe you can do it with `streamz.dataframe.DataFrame.aggregate/accumulate_partitions`. If you can share more about what the logic looks like and maybe generate a sample workflow, we can figure out something that works.
